docs/src/testing/debugging.md

```diff
@@ -1,13 +1,22 @@
 # Debugging
 
+> ℹ️ **Info**
+> To use `PrintTrait` you need to add `cheatcodes` package as a dependency in
+> your [`Scarb.toml`](https://docs.swmansion.com/scarb/docs/guides/dependencies#adding-a-dependency) 
+> using appropriate release tag.
+>```toml
+> [dependencies]
+> cheatcodes = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.4.0" }
+> ```
+
 Starknet Foundry standard library adds a utility method for printing inside tests to facilitate simple debugging.
 
 Here's a test with example use of print method:
 
-```cairo
+```rust
```

Comment:
Don't we mark all snippets as `cairo` code?

Reply:
Ok, I see that we usually use `rust`, but there are some places where we use `cairo`. Please unify that 🙏 
(prepare_cheatcode.md file)

Reply:
We do that in book for highlighting, I don't think there is a need to do that in design docs (especially the already closed ones)